snuba/datasets/configuration/storage_builder.py

```diff
@@ -81,13 +81,15 @@ def build_stream_loader(loader_config: dict[str, Any]) -> KafkaStreamLoader:
     ).from_kwargs()
     default_topic = Topic(loader_config["default_topic"])
     # optionals
-    pre_filter = (
-        CONF_TO_PREFILTER[loader_config[PRE_FILTER]["type"]](
+    pre_filter = None
+    if PRE_FILTER in loader_config and loader_config[PRE_FILTER] is not None:
+        pre_filter = StreamMessageFilter.get_from_name(
+            loader_config[PRE_FILTER]["type"]
+        ).from_kwargs(
             **loader_config[PRE_FILTER]["args"]
+            if loader_config[PRE_FILTER].get("args")
+            else {}
```

Comment:
This is a little cleaner

```suggestion
            loader_config[PRE_FILTER].get("args", {})
```